src/status_im/ui/components/bottom_bar/core.cljs

```diff
@@ -0,0 +1,129 @@
+   [status-im.ui.components.bottom-bar.styles :as tabs.styles]
+   [reagent.core :as reagent]
+   [status-im.ui.components.react :as react]
+   [status-im.utils.platform :as platform]
+   [status-im.ui.components.icons.vector-icons :as vector-icons]
+   [status-im.ui.components.common.common :as components.common]
+   [status-im.i18n :as i18n]
+   [status-im.ui.components.styles :as common.styles]
+   [re-frame.core :as re-frame]))
+
+(defn animate
+  ([visible duration to]
+   (animate visible duration to nil))
+  ([visible duration to callback]
+   (animation/start
+    (animation/timing visible
+                      {:toValue         to
+                       :duration        duration
+                       :useNativeDriver true})
+    callback)))
+
+(def tabs-list-data
+  [{:view-id             :chat-stack
+    :content             {:title (i18n/label :t/home)
+                          :icon  :main-icons/home}
+    :count-subscription  :chats/unread-messages-number
+    :accessibility-label :home-tab-button}
+   {:view-id             :wallet-stack
+    :content             {:title (i18n/label :t/wallet)
+                          :icon  :main-icons/wallet}
+    :count-subscription  :get-wallet-unread-messages-number
+    :accessibility-label :wallet-tab-button}
+   {:view-id             :profile-stack
+    :content             {:title (i18n/label :t/profile)
+                          :icon  :main-icons/profile}
+    :count-subscription  :get-profile-unread-messages-number
+    :accessibility-label :profile-tab-button}])
+
+(defn- tab-content [{:keys [title icon]}]
+  (fn [active? count]
+    [react/view {:style tabs.styles/tab-container}
+     [react/view
+      [vector-icons/icon icon (tabs.styles/tab-icon active?)]]
+     [react/view
+      [react/text {:style (tabs.styles/tab-title active?)}
+       title]]
+     (when (pos? count)
+       [react/view tabs.styles/counter-container
+        [react/view tabs.styles/counter
+         [components.common/counter count]]])]))
+
+(def tabs-list (map #(update % :content tab-content) tabs-list-data))
+
+(views/defview tab [view-id content active? accessibility-label count-subscription]
+  (views/letsubs [count [count-subscription]]
+    [react/touchable-highlight
+     (cond-> {:style    common.styles/flex
+              :disabled active?
+              :on-press #(re-frame/dispatch [:navigate-to-tab view-id])}
+       accessibility-label
+       (assoc :accessibility-label accessibility-label))
+     [react/view
+      [content active? count]]]))
+
+(defn tabs [current-view-id]
+  [react/view {:style tabs.styles/tabs-container}
+   (for [{:keys [content view-id accessibility-label count-subscription]} tabs-list]
+     ^{:key view-id} [tab view-id content (= view-id current-view-id) accessibility-label count-subscription])])
+
+(defn tabs-animation-wrapper [visible? keyboard-shown? tab]
+  [react/animated-view
+   {:style {:height    tabs.styles/tabs-height
```

Comment:
is there a reason why we need to use absolute position here? It causes problems with scroll views because if something appears on the screen (the connectivity bar for instance), the bottom of the scroll view offsets under the bar and the last items is partially hidden

Reply:
again can probably be done in another PR, it is already a bug and it get worst with the search bar PR

Reply:
it becomes `:absolute` only before animation (hiding of bootom tab when keyboard is shown), and yes it is necessary. Otherwise either animation looks ugly or it is slow and glitchy (because native driver can't be used)